graviti/dataframe/frame.py

```diff
@@ -25,6 +25,9 @@
 from graviti.dataframe.row.series import Series as RowSeries
 from graviti.utility.lazy import LazyFactory
 
+_MAX_REPE_NUMBER = 10
```

Comment:
`REPE` or `REPR`?
And which one is more accurate, `NUMBER` or `LENGTH` or `ROWS`?